packages/salesforce-adapter/src/change_validators/picklist_standard_field.ts

```diff
@@ -18,13 +18,22 @@ import {
 } from '@salto-io/adapter-api'
 import { collections } from '@salto-io/lowerdash'
 import { apiName, isCustom } from '../transformers/transformer'
-import { isPicklistField, isStandardValueSetPicklistField } from '../filters/value_set'
+import { isPicklistField, isValueSetReference } from '../filters/value_set'
+import { STANDARD_VALUE_SET } from '../filters/standard_value_sets'
+import { isInstanceOfType } from '../filters/utils'
+import { VALUE_SET_FIELDS } from '../constants'
 
 const { awu } = collections.asynciterable
 
+const isStandardValueSet = async (plField: Field): Promise<boolean> => {
+  const standardVSchecker = isInstanceOfType(STANDARD_VALUE_SET)
+  return isValueSetReference(plField)
+  && standardVSchecker(plField.annotations[VALUE_SET_FIELDS.VALUE_SET_NAME].value)
```

Comment:
a few comments on this:
1. (nit) IMO this is a little hard to read (especially since we don't have semicolons at the end of lines) - please either indent the 2nd line or add parentheses
```suggestion
  return isValueSetReference(plField)
    && standardVSchecker(plField.annotations[VALUE_SET_FIELDS.VALUE_SET_NAME].value)
```
```suggestion
  return (
    isValueSetReference(plField)
    && standardVSchecker(plField.annotations[VALUE_SET_FIELDS.VALUE_SET_NAME].value)
  )
```
2. seems we're "losing" a chance to use safe extraction logic with `.annotations[VALUE_SET_FIELDS.VALUE_SET_NAME]` here - maybe add a `getValueSetReference` so that we know it exists? if not, we should allow for the chance that it becomes undefined (use `?.` instead of `.`)
3. I'm not sure we can assume that the reference's `value` exists at this point - I think you should use `getResolvedValue` instead

Reply:
@netama AFAIK currently everything in the adapter (including change validators) get fully resolved elements

Reply:
got it, so ignore 3 :)

Reply:
re 2 - the `isValueSetReference(plField)` check assueres that the field has this annotation, so i guess it's ok to just access it in the right part of the `&&`